ci/jenkins/README.md

```diff
@@ -1 +1,97 @@
 # Nephe CI: Jenkins
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Jenkins Deployment](#jenkins-deployment)
+  - [Teraform](#teraform)
+  - [Ansible](#ansible)
+  - [Jenkins Job](#jenkins-job)
+    - [Requirements](#requirements)
+    - [Apply the jobs](#apply-the-jobs)
+    - [Jenkins job updater](#jenkins-job-updater)
+- [List of jenkins job](#list-of-jenkins-job)
+<!-- /toc -->
+
+## Introduction
+
+This directory includes all the scripts required to run CI for nephe.
+
+We have a static VM configured to run as jenkins node.  When a jenkins job is
+triggered, the nephe code will be cloned on the static VM. The jenkins node will
+call terraform to a deploy dynamic VM and use ansible for installing any missing
+packages like docker. Then tests will be run on the dynamic VM, the VM will be
+destroyed once the test succeeds or fails.
+
+## Jenkins Deployment
+
+### Teraform
+
+We create terrform tfvars file in nephe-ci.sh with jenkins credetials. [deploy.sh](./deploy.sh)
```

Comment:
Rephrase: Nephi-ci.sh (link) is responsible for creating terraform tfvars files.

Reply:
done